crates/sui-indexer/tests/integration_tests.rs

```diff
@@ -2,7 +2,6 @@
 // SPDX-License-Identifier: Apache-2.0
 
 // integration test with standalone postgresql database
-#[cfg(feature = "pg_integration")]
```

Comment:
why remove this? This will trigger the integration test in the CI rust test(as opposed to CI Integration test).

Reply:
sry that was a typo